docker/Dockerfile.bionic

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+# docker build -f docker/Dockerfile.bionic .
+
```

Comment:
@jankoprowski is this simply a somewhat modified version of nushell/nu-base? Do we really want to maintain two Dockerfiles (that are very similar) when you could just use one as a base and then add additional dependencies?

Reply:
Also, having this in the "docker" folder and with tag/extension "bionic" doesn't make it clear that it's for the debian packaging. Since I just put in #663 to update the base to 18.04, perhaps we could have a subfolder in docker for debian releases, and then a Dockerfile that gets rid of most of the content, but just keeps the additions. For example:

```
ARG FROMTAG=latest
FROM quay.io/nushell/nu-base:${FROMTAG}
RUN apt-get update && apt-get install -y
    devscripts \
    debhelper

RUN rustc -Vv && cargo build --release && \
    debuild -b -us -uc -i && \
    dpkg -i ../nu_0.2.0-1_amd64.deb && \
    chsh -s /usr/bin/nu && \
    echo 'ls | get name | echo $it' | /usr/bin/nu
```
I'm not familiar with packaging, but you could of course modify the last lines to isolate those that can more easily be cached.

Reply:
We would want the developer / user inspecting the repository to clearly know that this Dockerfile is for packaging, and the others are for user-facing images.

Reply:
Really good points. Maybe docker/packaging directory?

Reply:
Yes! docker/packaging is even more clear, it would be intuitive to see:

```
docker/
   packaging/
      Dockerfile.bionic
```
And easily infer that (given you are familiar with ubuntu) the file is intended for the bionic distribution. The only more clear label would be something like:

```
docker/
   packaging/
      Dockerfile.ubuntu-bionic
```
If there could be plans for other distributions, e.g.,

```
docker/
   packaging/
      Dockerfile.ubuntu-bionic
      Dockerfile.ubuntu-xenial
      Dockerfile.debian-jessie
```
Yes, the reason for it being so big is that it has all of the cargo deps for nu already there, along with the source code. The size of the image that you build from the recipe above (I would guess?) should be similar!

Reply:
> ```
> ARG FROMTAG=latest
> FROM quay.io/nushell/nu-base:${FROMTAG}
> RUN apt-get update && apt-get install -y
>     devscripts \
>     debhelper
> 
> RUN rustc -Vv && cargo build --release && \
>     debuild -b -us -uc -i && \
>     dpkg -i ../nu_0.2.0-1_amd64.deb && \
>     chsh -s /usr/bin/nu && \
>     echo 'ls | get name | echo $it' | /usr/bin/nu
> ```

This does not work because There is no debian directory in /code. Any ideas?